osf/migrations/0133_merge_20180918_1436.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-09-18 14:36
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
Not sure why there were two merge migrations created. Would you mind updating with develop and "rebasing" the migration (i.e. putting it at the end)?